css-brochure-form/styles.css

```diff
@@ -0,0 +1,62 @@
+h1 {
+  text-align: center;
+  color: #f07167;
+  font-family: AtlasGrotesk, "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  src: url("AtlasGrotesk-Light-Web.woff2") format("woff2");
+  text-transform: uppercase;
+}
+
+body {
+  text-align: center;
+}
+
+section {
+  font-family: "Brandon Grotesque", AtlasGrotesk, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+}
+
+input.first,
+input.second,
+input.third {
+  border-radius: 6px;
+  margin-top: 5px;
+  padding: 12px 25rem 12px 8px;
+}
+
+input.fourth {
+  border: 6px;
+  background-color: #f07167;
+  color: white;
+  padding: 15px 30px;
+  border-radius: 50px;
+  text-transform: uppercase;
+  margin-left: 30rem;
```

Comment:
you want to avoid using large amounts of margin to position elements. It would be better to wrap the form in a `container` element, set the width of that element to about 600px, then using `margin: 0 auto` to center that container on the page. 

Then you can just do a little nudge here and there.